kernel/kernel-idgenerator-service/src/main/java/io/mosip/kernel/idgenerator/constant/HealthConstants.java

```diff
@@ -1,6 +1,6 @@
-package io.mosip.kernel.uingenerator.constant;
+package io.mosip.kernel.idgenerator.constant;
 
-public class UINHealthConstants {
+public class HealthConstants {
```

Comment:
Why are we doing this renaming ?

Reply:
There are two classes, UINHealthConstants and VIDHealthConstants, both are having exactly same constants defined. This caused increased percentage of duplication in sonarcloud report. 
To avoid the duplications, I changed it to common class. 
If this is not suggestable, I'll rollback the change.

Reply:
Rolled back as discussed, added the files again.